### ISSUE TITLE
traceback if run w/ no args

### ISSUE THREAD

**Issue by jamescassell** [shepdelacreme/benchparse#5]
Would expect some error, or usage text instead.  Also, maybe it's an issue w/ the CIS parsing code?

**Comment by shepdelacreme** [comment]
Yeah...I don't think it is anything to do with the CIS parsing code, just an issue with my python :) 

Can you provide the command as run?

I'm assuming you literally ran it without any arguments, which means you didn't specify the benchmark file input `-X` or `--xccdf`. That argument should be marked `required=True` or I guess it should maybe change to be a positional instead of an optional argument type since it isn't _optional_. 

Without a benchmark file, it will bomb out in the class `__init__` function since it tries to open that file without any exception handling code.

**Comment by jamescassell** [comment]
yeah, it was literally `python ~/src/benchparse.py` or `chmod +x src/benchparse.py ; src/benchparse.py`

Sounds like it was just user error, but I did waste a few minutes on it.
